apps/website/src/data/animal-quest.ts

```diff
@@ -0,0 +1,98 @@
+import type ambassadors from "@/data/ambassadors";
+
+type Ambassadors = keyof typeof ambassadors;
```

Comment:
I pushed a commit to your branch. Feel free to drop it, but that seemed to be the easiest for me to suggest the changes:
- use satisfies to type check the data first
- widen the type back to `Record<string, Ambassador>` for the default export so the existing code does not have to change (for example it would complain there is not plush.soon when using the narrower type)
- use a type guard (`isAmbassadorKey`) in `getAmbassadorEpisode` to narrow the type of the key while allowing strings (`AmbassadorKey | string`) so we can keep using the getter without changing the code but we gain auto completion when calling the getter